builtin/check-ref-format.c

```diff
@@ -53,31 +56,28 @@ static int check_ref_format_branch(const char *arg)
 
 int cmd_check_ref_format(int argc, const char **argv, const char *prefix)
 {
-	int i;
-	int normalize = 0;
+	enum {
+		CHECK_REF_FORMAT_BRANCH,
+	};
+	int i = 0;
```

Comment:
I think it would make more sense to get rid of `i` now. Its role was to iterate through the command-line options, which is now no longer performed in this function.